core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -224,6 +225,63 @@ public void adddate() {
     assertEquals(missingValue(), eval(dsl.adddate(nullRef, missingRef)));
   }
 
+  @Test
+  public void convert_tz() {
```

Comment:
Could you make a test fixture for convert_tz? Each little assert block is a separate test case.